DEVELOPMENT_ENVIRONMENT.md

```diff
@@ -21,40 +21,54 @@ You can find the installation instructions for your distribution [here](https://
 
 ### Docker on Mac OSX (and Windows, just in case)
 
-Install boot2docker as per the instructions that can be found [here](http://boot2docker.io/).
-After installing boot2docker, make sure to initialize the boot2docker VM by running:
+Install the docker toolbox as per the instructions that can be found [here](https://www.docker.com/toolbox). The toolbox will install virtualbox, docker, docker-compose, and docker-machine.
+After installing the toolbox, make sure to initialize the boot2docker VM by running:
 
 ```bash
-$ boot2docker init
+$ docker-machine create -d virtualbox default
 ```
 
-## Fig
+### Docker on Ubuntu
 
-Install Fig as described in the instructions [here](http://www.fig.sh/install.html)
+Run the following commands and you can skip everything related to docker-machine in the instructions since Ubuntu is a Linux distro which can run containers on bare metal.
```

Comment:
This sentence does not parse well because its too long, and might confuse the user rather than help them. It's like a dreaded "goto" statement.

There is already have an "if" statement at the beginning of the "Docker Machine Initialization Pre-requisites" that should signal to Ubuntu users not to do whatever is described there. That should be sufficient.